pandas/_testing/asserters.py

```diff
@@ -1344,6 +1344,7 @@ def assert_frame_equal(
                 rtol=rtol,
                 atol=atol,
                 check_index=False,
+                check_flags=False,
```

Comment:
well shouldn't we acutally pass the `check_flags` thru?

Reply:
We check the flags on the DataFrame, so no need to check for every column